PatientWebApplication/HealthClinic.DAL/Dtos/MessageDto.cs

```diff
@@ -8,19 +8,22 @@ public class MessageDto
 
         public DateTime TimeStamp { get; set; }
 
+        public string DateStamp { get; set; }
+
         public string PharmacyName { get; set; }
 
         public string DateAction { get; set; }
 
-        public MessageDto(string text, DateTime timestamp, string pharmacyname, string dateaction)
+        public MessageDto(string text, string datestamp, DateTime timestamp, string pharmacyname, string dateaction)

```

Comment:
Chnage names of variables to match lines above, for example datestamp should be dateStamp, pharmacyname should be pharmacyName...